README.md

```diff
@@ -38,7 +38,7 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```There are global and local scopes.  Any variable created/defined globally does not need var and can be accessed by any function.  Any var or function created locally can only be accessed by functions within that scope.  
```

Comment:
What you have here is correct. Just to clarify, omitting the var keyword has the effect of creating that variable in global scope but even when declaring global variables it is best practice to declare with the var keyword.